### DISCUSSION TITLE
::selection background-color applies globally

### DISCUSSION THREAD

**Discussion by Kellaro** [codex-team/editor.js#1931] [Q&A]
Stumbled upon ::selection styles, which are applied globally throughout whole app. Is it by design, or a bug?

**Comment by philipaarseth** [comment]
Would appreciate if this was removed asap. I don't see any reason why it would make sense to apply this (or anything for that matter) globally.

**Comment by Mdev303** [comment]
still a problem

**Comment by cpjhong** [comment]
still a problem

**Comment by itspers** [comment]
this is crazy

**Comment by neSpecc** [comment]
The problem is [fixed](https://github.com/codex-team/editor.js/pull/2430) in upcoming 2.28. The release candidate will be published in next few days
